sparkmagic/sparkmagic/tests/test_handlers.py

```diff
@@ -0,0 +1,134 @@
+from mock import MagicMock
+from nose.tools import with_setup, raises, assert_equals, assert_is
+
+from sparkmagic.serverextension.handlers import ReconnectHandler
+from sparkmagic.kernels.kernelmagics import KernelMagics
+
+
+reconnect_handler = None
+session_manager = None
+kernel_manager = None
+individual_kernel_manager = None
+client = None
+session_list = None
+spark_events = None
+path = 'some_path.ipynb'
+kernel_id = '1'
+username = 'username'
+password = 'password'
+endpoint = 'http://endpoint.com'
+response_id = '0'
+good_msg = dict(content=dict(status='ok'))
```

Comment:
Any particular reason you are doing this instead of just `{'content': {'status': 'ok'}}`?  This doesn't seem like the typical Python style to me.